### ISSUE TITLE
Feature Request: Add option to suppress alert in testing

### ISSUE THREAD

**Issue by yakkomajuri** [zacanger/alert-node#7]
First of all, thanks for the work! The package helps me a lot!

I'm building a website with Node.js and have recently come across the "issue" that when I run mocha tests, alerts fire on top of the code editor which is rather annoying. I'm using alerts to express errors in the user registration process, which I test quite thoroughly, and, as a result, get a bunch of alerts from it that I have to manually close.

I've fixed the issue for myself with my own little wrapper:

```
const alert = require('alert');

var generateAlert = module.exports = function(msg) {
    if (process.env.NODE_ENV == "development") {
        alert(msg);
    }
}
```
When testing, I set `NODE_ENV ` to `test` which suppresses the alert.

So I was thinking of building a feature where a user can pass options to `alert()` to determine in which environments alert should or should not fire. `thingToUse` could also be an option, and not setting the `options` parameter would of course not change anything.

Something like:

`alert("Username already in use", { thingToUse: "yad", env: ["development", "production"]})`

It would then read from `process.env.NODE_ENV` to find the value set by the user.

Wondering if this is of interest or if it just adds unnecessary size to the package. If it is, I'll get on it right away!

Cheers!

**Comment by zautumnz** [comment]
Rather than adding another option, would an env var work? Something like `DISABLE_ALERT`, which would be used like `NODE_ENV=test DISABLE_ALERT=1 jest`? I feel like that would be less of a burden on other users, especially if they have a bunch of `NODE_ENV`s, or might want to disable in other cases with the same `NODE_ENV`.

**Comment by yakkomajuri** [comment]
Even better! Don't know how that didn't come to mind. Should I do that? I guess it's just a matter of wrapping it all in an if statement

**Comment by zautumnz** [comment]
Sure if you'd like to! The check can probably go at the bottom of `node.js`, just conditionally exporting either `getAlert` or a noop.

**Comment by yakkomajuri** [comment]
Got you!

**Comment by yakkomajuri** [comment]
Submitted the PR - just added a ternary operator to the last line in `node.js` and a line in README about it

**Comment by zautumnz** [comment]
Released in v4.1.0
